fsspec/caching.py

```diff
@@ -426,8 +426,9 @@ def _fetch(self, start, end):
 class KnownPartsOfAFile(BaseCache):
     name = "parts"
 
-    def __init__(self, blocksize, fetcher, size, data={}, **_):
+    def __init__(self, blocksize, fetcher, size, data={}, strict=True, **_):
```

Comment:
Should add minimal docs for these arguments

Reply:
(or just refer to the parquet calling function, which I see does have the doc)

Reply:
Agree - I added a new docstring to KnownPartsOfAFile